quickstart/101-vm-with-infrastructure/variables.tf

```diff
@@ -8,4 +8,11 @@ variable "resource_group_name_prefix" {
   type        = string
   default     = "rg"
   description = "Prefix of the resource group name that's combined with a random ID so name is unique in your Azure subscription."
+}
+
+
+variable "username" {
+  type        = string
+  description = "The username for the local account that will be created on the new VM."
+  default     = "azureadmin"
```

Comment:
The default value is different than the corresponding default value in `coalesce` function.

Reply:
Fixed.